### PR TITLE
Navbar and SideBar Design Changes

### PR DESCRIPTION
**Pull request by saadiqbal-dev**
—BOS-Workspace-Docs
1. Navbar/Sidebar Design Changes
2. Content Design Updates/Settings Page Updates
3. White Label Customization Changes (Classes moved to Index, links in Navbar etc)
4. Mobile Responsiveness
5. Embeddings in Markdown

--Styling Video
https://github.com/NEARBuilders/bos-workspace-docs/assets/165388405/5f839f53-a42d-442f-83de-640c3f0701cd
--Responsiveness/Customization
https://github.com/NEARBuilders/bos-workspace-docs/assets/165388405/8ff98813-30f0-488f-9b8e-517d48f2a493
--Embeddings in Markdown
https://github.com/NEARBuilders/bos-workspace-docs/assets/165388405/ed5b25b4-6da4-4f93-a76f-da9ce39c50f2

### CODE REVIEW

File: widget/index.jsx [hunk 1]
```diff
@@ -86,10 +86,17 @@ const config = {
   },
 };
 
+const InterCSS = fetch(
```
Comment (elliotBraem) [inline widget/index.jsx]: This fetch being in our root makes the whole page reload on navigation. We'll need to incorporate it into the layout or find another solution. 

But for now; I propose not importing a font, just use web fonts

File: widget/components/Sidebar.jsx [hunk 2]
```diff
@@ -21,31 +21,75 @@ Object.keys(documents).forEach((path) => {
   }
 });
 
+const Sidebar = styled.div`
```
Comment (elliotBraem) [inline widget/components/Sidebar.jsx]: I think for the ability to customize styles in other projects, for a white label solution, this CSS should be moved to `index`, where "sidebar" className was defined before

File: widget/components/Header.jsx [hunk 3]
```diff
@@ -1,61 +1,72 @@
...
+};
 
-return (
-  <div className="header">
-    <div className="branding">
-      <Link to={`/${basePath}`} className="link">
-        <h3>everything.dev</h3>
-      </Link>
-    </div>
+const Header = styled.div`
```
Comment (elliotBraem) [inline widget/components/Header.jsx]: This too, replace with class names in `index`

File: md/getting_started/index.md [hunk 4]
```diff
@@ -11,29 +11,36 @@
...
+
 ```js
 npm install bos-workspace
 ```
+
 To verify `bos-workspace` in installed, you can check for a current version:
+
 ```js
-bos-workspace -V
+bos - workspace - V;
```
Comment (itexpert120) [inline md/getting_started/index.md]: fix it, `bos-workspace -V`

File: widget/MarkdownView.jsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+`;
+const Embedded = styled.span`
+  white-space: normal;
+  p {
+    white-space: normal;
+  }
+`;
+function MarkdownViewer(props) {
+  const onLink = (props) => {
+    console.log("onLink", props);
```
Comment (itexpert120) [inline widget/MarkdownView.jsx]: remove console log

File: widget/MarkdownView.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            accountId: accountId.toLowerCase(),
+            hideAccountId: true,
+            tooltip: true,
+          }}
+        />
+      </span>
+    ));
+  return (
+    <Wrapper>
+      {/* <Markdown text={props.value} onMention={renderMention} /> */}
```
Comment (itexpert120) [inline widget/MarkdownView.jsx]: remove this comment

File: widget/components/Sidebar.jsx [hunk 7]
```diff
@@ -21,31 +23,39 @@ Object.keys(documents).forEach((path) => {
   }
 });
 
+const Sidebar = styled.div``;
+
+console.log(documents);
```
Comment (itexpert120) [inline widget/components/Sidebar.jsx]: remove this log

File: widget/document.jsx [hunk 8]
```diff
@@ -2,9 +2,15 @@ const { basePath, param, _params } = props;
 
 let adapter = "${config_account}/widget/utils.db"; // this could come from props
 
-const Template = VM.require("${config_account}/widget/PR.Template");
+const Template = VM.require("${config_account}/widget/PR.Template") || {
+  Template: () => <></>,
+};
 
-const { get } = VM.require(adapter);
+console.log("PARAMS", _params);
```
Comment (itexpert120) [inline widget/document.jsx]: remove this log

File: widget/components/Content.jsx [hunk 9]
```diff
@@ -0,0 +1 @@
+const Container = styled.div``;
```
Comment (itexpert120) [inline widget/components/Content.jsx]: remove this unused file

File: widget/components/Header.jsx [hunk 10]
```diff
@@ -1,61 +1,169 @@
...
+    groupedSections[parentSection] = [];
+  }
+
+  // Ensure child section is only added once per parent section
+  if (childSection && !groupedSections[parentSection].includes(childSection)) {
+    groupedSections[parentSection].push(childSection);
+  }
+});
+
+const NearIcon = () => {
```
Comment (itexpert120) [inline widget/components/Header.jsx]: move to it's own file to remove clutter from Header component

File: widget/home.jsx [hunk 11]
```diff
@@ -1,14 +1,16 @@
 const LandingPageContainer = styled.div`
   display: flex;
   flex-direction: column;
-  align-items: center;
   justify-content: center;
+  font-family: Inter;
```
Comment (itexpert120) [inline widget/home.jsx]: always add sans-serif fallback to font-family

File: widget/edit.jsx [hunk 12]
```diff
@@ -1,4 +1,4 @@
-const { MarkdownViewer } = VM.require("${alias_devs}/widget/markdown.view") || {
+const { MarkdownViewer } = VM.require("docs.bos-workspace.near/widget/MarkdownView") || {
```
Comment (itexpert120) [inline widget/edit.jsx]: replace docs.bos-workspace.near with ${config_account}

File: widget/settings.jsx [hunk 13]
```diff
@@ -35,17 +46,29 @@ const PreviewContent = styled.div`
   font-size: 16px;
 `;
 
-const Select = styled.select``;
-
 const Option = styled.option``;
```
Comment (itexpert120) [inline widget/settings.jsx]: unused

File: widget/settings.jsx [hunk 14]
```diff
@@ -35,17 +46,29 @@ const PreviewContent = styled.div`
...
 
 const Label = styled.label`
```
Comment (itexpert120) [inline widget/settings.jsx]: unused

**Review by itexpert120 — COMMENTED** [review]
Resolve all these changes

**Pull request closed without merging**